### PR TITLE
fix(module:pagination): add ul tag

### PR DESCRIPTION
**Pull request by chenc041**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

The pagination component missing `ul` tag
Issue Number:  [#7493](https://github.com/NG-ZORRO/ng-zorro-antd/issues/7493)

Close:  [#7493](https://github.com/NG-ZORRO/ng-zorro-antd/issues/7493)

## What is the new behavior?
Add `ul` tag

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/pagination/pagination-default.component.ts [hunk 1]
```diff
@@ -38,40 +38,42 @@ import { PaginationItemRenderContext } from './pagination.types';
...
-        [nzSize]="nzSize"
-        [showSizeChanger]="showSizeChanger"
-        [showQuickJumper]="showQuickJumper"
-        [pageIndex]="pageIndex"
-        [pageSize]="pageSize"
-        [pageSizeOptions]="pageSizeOptions"
-        (pageIndexChange)="onPageIndexChange($event)"
-        (pageSizeChange)="onPageSizeChange($event)"
-      ></div>
+      <ul>
```
Comment (simplejason) [inline components/pagination/pagination-default.component.ts:41]: Add `ant-pagination` to `ul` tag, and remove this class from `nz-pagination`
```diff
- <ul>
+ <ul class='ant-pagination'>
```

**Review by simplejason — CHANGES_REQUESTED** [review]

File: components/pagination/pagination-default.component.ts (hunk 1 above)
Comment (chenc041) [reply components/pagination/pagination-default.component.ts]: Has been solved

**Comment by simplejason** [comment]
If you don't mind, I'll fix the testing codes and then we can merge it :)

**Comment by chenc041** [comment]
> If you don't mind, I'll fix the testing codes and then we can merge it :)

ok~

**Comment by chenc041** [comment]
I submitted for the first time, all of the ci are passed, but after a period of time not line, this is to have a change?

**Comment by simplejason** [comment]
> I submitted for the first time, and all of the ci is passed, but after a period of time, not line, this is to have a change? @simplejason

I think it is caused by the className `.ant-pagination`, I will keep it on `nz-pagination` for now. Previously I expected the same dom as ant-design, but it breaks the existing tests, so I will rollback it for now :)

**Comment by chenc041** [comment]
> > I submitted for the first time, and all of the ci is passed, but after a period of time, not line, this is to have a change? @simplejason
> 
> I think it is caused by the className `.ant-pagination`, I will keep it on `nz-pagination` for now. Previously I expected the same dom as ant-design, but it breaks the existing tests, so I will rollback it for now Of course you are welcome to submit code if you wish :)
> 
> ```
> 1. add `class: 'ant-pagination'` to `host` of `nz-pagination`
> 2. remove `.ant-pagination` from the ul tag of `nz-pagination-default` and `nz-pagination-simple`
> ```
> 
> update: done

thks

**Comment by simplejason** [comment]
@chenc041 After CI passed, I will merge it and it will be released with `ng-zorro-antd@14.0.0` in this week, thanks for your contributions

**Comment by chenc041** [comment]
> @chenc041 After CI passed, I will merge it and it will be released with `ng-zorro-antd@14.0.0` in this week, thanks for your contributions

ok~
You are welcome

**Review by simplejason — APPROVED** [review]
LGTM

**Pull request merged**